test/dynamo/test_higher_order_ops.py

```diff
@@ -1292,6 +1292,47 @@ def forward(self, getitem, const):
     return (sin,)""",
             )
 
+    def test_map_example_value_metadata_consistent_with_eager(self):
+        from torch._higher_order_ops.map import map_dense
+
+        backend = EagerAndRecordGraphs()
+
+        def inner(x):
+            return x.sin(), x.cos().T, x.sin().view(-1)
+
+        inps = [
+            torch.randn(3),
+            torch.randn(3, 4),
+            torch.randn(3, 4, 5, requires_grad=True),
+        ]
```

Comment:
This seems very insufficient for testing strangely strided inputs?

Reply:
added more strangely strided inputs